Spoon Master/Controller/DetailViewController.swift

```diff
@@ -7,10 +7,63 @@
 //
 
 import UIKit
-
-final class DetailViewController: UIViewController {
-
+import Then
+import Kingfisher
+    
+final class DetailViewController: UIViewController, UICollectionViewDelegate {
+            
+    var data: Recipe!
+    
+    @IBOutlet private weak var avatarFood: UIImageView!
+    @IBOutlet private weak var summary: UITextView!
+    @IBOutlet private weak var nameFood: UILabel!
+    @IBOutlet private weak var money: UILabel!
+    @IBOutlet private weak var time: UILabel!
+    @IBOutlet private weak var score: UILabel!
+    @IBOutlet weak var button: UIButton!
```

Comment:
```suggestion
    @IBOutlet private weak var button: UIButton!
```

button gì ghi rõ ra, có thể favoriteButton